CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.4.3
+
+* Fixed bug in HTTP errors processing.
```

Comment:
We use the present tense in other log messages. And also, maybe `in HTTP error translations` (to match corresponding decorator's name)